### PR TITLE
Update phenotype pipeline CGC import

### PR DESCRIPTION
**Pull request by dglemos**
Update import for source CGC:
- the URL to download the input is different
- input format is different
- save more data into phenotype_feature_attrib: [ENSVAR-3591](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-3591) this change requires we add 3 more attrib types into the attrib_type table.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm [hunk 1]
```diff
@@ -76,60 +78,45 @@ sub fetch_input {
...
-      unlink($workdir."/".$file_opent) if -e $workdir."/".$file_opent;
-      print $errFH "WARNING: No OpenTarget file found for current month ($file_opent), will check past month.\n";
-    } elsif ($resHTTPcode == 200){ #Successful
-      print $errFH "HTTP code 200 (successful) but file ($file_opent) is missing \n" unless -e $workdir."/".$file_opent;
-    }
-    if (-e $workdir."/".$file_opent) { $found = 1;}
+  print $logFH "INFO: Found input file ($file_opent)\n" ;
+  my $valid_file = $self->validate_input_file($file_opent);
+  if(!$valid_file) {
+    print $errFH "Input file ($file_opent) has incorrect format\n";
```
Comment (olaaustine) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm:108]: Suggesting error message should be 
"Input file ($file_opent) has incorrect format or empty \n";

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm [hunk 2]
```diff
@@ -76,60 +78,45 @@ sub fetch_input {
...
+    die ("Input file ($file_opent) has incorrect format\n");
```
Comment (olaaustine) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm:109]: Suggesting error message should be 
"Input file ($file_opent) has incorrect format or empty \n";

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm [hunk 3]
```diff
@@ -164,24 +151,72 @@ sub run {
...
+
+=cut
+
+sub validate_input_file {
+  my ($self, $input_file) = @_;
+
+  my $valid = 1;
+
+  # empty file
+  if(-z $input_file) {
```
Comment (olaaustine) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm:180]: Suggesting you change the $input_file to $self->workdir."/".$input_file

Comment (dglemos) [reply modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm]: Done. Thanks for noticing it.

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm (hunk 2 above)
Comment (dglemos) [reply modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm]: Done!

**Review by olaaustine — APPROVED** [review]

**Comment by olaaustine** [comment]
Merged into main and release/109

**Pull request merged**